### PR TITLE
daemon, overlord/state: expose individual changes

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Comment by pedronis** [comment]
LGTM

**Comment by mvo5** [comment]
Looks good

File: overlord/state/state.go [hunk 1]
```diff
@@ -277,6 +277,12 @@ func (s *State) Changes() []*Change {
 	return res
 }
 
+// Change returns the change for the given ID.
+func (s *State) Change(id string) *Change {
+	s.reading()
```
Comment (niemeyer) [inline overlord/state/state.go]: The method entrance test should get a new line on the reading side.

**Comment by niemeyer** [comment]
LGTM

**Pull request merged**